docs/snippets/multiple-confirmation-notifications.md

```diff
@@ -0,0 +1,2 @@
+!!! warning "Multiple notifications for blocks"
+    Depending on the node setup and sync status, multiple confirmation notifications for the same block hash may be sent by a single tracking mechanism. Both the WebSocket and HTTP callback options could independently send multiple confirmation notices. In order to prevent potential issues, integrations must track these block hashes externally to the node and prevent any unwanted actions based on multiple notifications.
```

Comment:
Depending on the node setup and sync status, confirmations may be seen multiple times across both WebSocket and HTTP callback options. In order to prevent potential issues, integrations must track these block hashes externally to the node and prevent any unwanted actions based on multiple notifications.

Alternative wording